pytask_io/exceptions.py

```diff
@@ -0,0 +1,8 @@
+class PyTaskIOException(Exception):
```

Comment:
This is a separate issue that i am working on, please remove this custom exception